docs/dls/component-design-guidelines.md

```diff
@@ -103,6 +103,8 @@ accessibility.
 [7]: https://v4.mui.com/components/cards/
 [8]: https://v4.mui.com/customization/palette/#default-values
 [9]: https://v4.mui.com/customization/typography/
-[10]: https://backstage.io/docs/getting-started/app-custom-theme
-[11]: https://backstage.io/docs/getting-started/app-custom-theme#overriding-backstage-and-material-ui-components-styles
+[10]: https://backstage.io/docs/conf/user-interface
+
+<!-- [11]: https://backstage.io/docs/getting-started/app-custom-theme#overriding-backstage-and-material-ui-components-styles -->
```

Comment:
Should we just delete this? Them the item currently on line 110 would go from `12` to `11`?

Reply:
Done